examples/dtls-echo/README.md

```diff
@@ -29,49 +25,42 @@ Do not forget to copy the IPv6 addresses!
 For the client:
 
     PORT=tap0 make term
-    dtlsc <IPv6's server address> "DATA TO DATA TO DATA!"
+    dtlsc <IPv6's server address[%netif]> "DATA to send under encrypted channel!"
 
 # Testings
 ## Boards
 
-Those boards that do not support  the `../gnrc_networking` example are included
-in the `BOARD_INSUFFICIENT_MEMORY`, plus the  board `cc2650stk`.
+Boards that do not support the `../gnrc_networking` example are included
+in the `BOARD_INSUFFICIENT_MEMORY`, plus the board `cc2650stk`.
 
 There are certain boards that are having issues with `crypto.c` and
 `dtls_time.h` Which for now are in the the `BOARD_BLACKLIST`.
 
 The boards that requires `periph_conf.h` are not tested.
```

Comment:
Unrelated but this line makes no sense IMHO.

Reply:
I think that you are right with the line 39, but also applies to the lines 36-37. I'll remove them.